fix-point/src/Refactor.hs

```diff
@@ -0,0 +1,39 @@
+module Rename where 
+
+import qualified Data.Text as Universum
+import qualified Data.Text.IO as Universum
+
+changeLine :: Universum.Text -> Universum.Text -> Universum.Text 
```

Comment:
Usually `text` is imported like this:

```haskell
import Data.Text (Text)

import qualified Data.Text as T
```

So you can use `Text` type without qualification.